src/render/OpenGL.cpp

```diff
@@ -455,7 +460,8 @@ void CHyprOpenGLImpl::renderTextureInternalWithDamage(const CTexture& tex, wlr_b
         glUniform1i(shader->applyTint, 0);
     }
 
-    glVertexAttribPointer(shader->posAttrib, 2, GL_FLOAT, GL_FALSE, 0, fullVerts);
+    // TODO: ???
+    // glVertexAttribPointer(shader->posAttrib, 2, GL_FLOAT, GL_FALSE, 0, fullVerts);
```

Comment:
dunno. That's how others do it.

Reply:
Just look 10 lines further below there is an if-else block that sets it anyways

Reply:
done